tests/x11/yast2_lan_restart_devices.pm

```diff
@@ -42,7 +42,7 @@ sub add_device {
         send_key 'alt-g';             # General
         send_key 'alt-i';             # Bridged devices
         assert_screen 'yast2_lan_bridged_devices';
-        if (check_screen('yast2_lan_default_NIC_bridge')) {
+        if (check_screen('yast2_lan_default_NIC_bridge', 0)) {
```

Comment:
Same here, it may break the test, and it's not goal of this PR. So if you trigger 10-50 runs for each of such changes, I'm fine with it, otherwise I would handle it separately. What's about creating progress ticket with all these test suites mentioned?

Reply:
there is an assert_screen immediately in before. This will work

Reply:
Agreed, risk is pretty low